flow360/component/simulation/models/bet/bet_translator_interface.py

```diff
@@ -979,6 +992,7 @@ def parse_xrotor_file(xrotor_file_content):
     xrotor_input_dict["cdmin"] = [0] * n_aero_sections
     xrotor_input_dict["clcdmin"] = [0] * n_aero_sections
     xrotor_input_dict["dcddcl2"] = [0] * n_aero_sections
+    xrotor_input_dict["dcddcm2"] = [0] * n_aero_sections
```

Comment:
```suggestion
    xrotor_input_dict["dcddcm2"] = [0] * n_aero_sections  # currently unused by BET translator but we are recording it in case we need it in the future.
```